common/logisticspipes/gui/guidebook/GuiGuideBookTexturedButton.java

```diff
@@ -0,0 +1,104 @@
+package logisticspipes.gui.guidebook;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+
+import com.sun.jna.platform.win32.WinUser;
+import lombok.Setter;
+
+import logisticspipes.LPConstants;
+
+public class GuiGuideBookTexturedButton extends GuiButton {
+
+	private int u0, v0, u1, v1, sizeX, sizeY;
+	@Setter
+	private boolean hasDisabledState;
+	@Setter
+	private EnumButtonType type;
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	public GuiGuideBookTexturedButton(int buttonId, int x, int y, int widthIn, int heightIn) {
+		super(buttonId, x, y, widthIn, heightIn, "");
+		this.u0 = 0;
+		this.v0 = 0;
+		this.u1 = 0;
+		this.v1 = 0;
+		this.sizeX = 0;
+		this.sizeY = 0;
+		this.hasDisabledState = false;
+		this.type = EnumButtonType.TAB;
```

Comment:
Instead of setting every field here again, call `this(buttonId, x, y, widthIn, heightIn, 0, 0, …)` instead

Reply:
Done